gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -418,17 +418,23 @@ def CycleDisplayItem(direction):
 def OpenItemInfoWindow (btn, slot):
 	pc = GemRB.GameGetSelectedPCSingle ()
 
-	slot_item = GemRB.GetSlotItem (pc, slot)
-	item = GemRB.GetItem (slot_item["ItemResRef"])
+	slotItem = GemRB.GetSlotItem (pc, slot)
+	slotType = GemRB.GetSlotType (slot, pc)
+
+	# PST: if the slot is empty but is also the first quick weapon slot, display the info for the "default" weapon
+	if GameCheck.IsPST() and slotItem is None and slotType["ID"] == 10 and GemRB.GetEquippedQuickSlot(pc) == 10:
+		slotItem = GemRB.GetSlotItem (pc, 0)
```

Comment:
You can call DisplayItem here and return early, since none of the other code is relevant for these items.